Sources/Controllers/TargetMenu/OATargetMenuViewController.mm

```diff
@@ -640,7 +640,10 @@ - (void) createMapDownloadControls
     if (_localMapIndexItem)
     {
         self.downloadControlButton = [[OATargetMenuControlButton alloc] init];
-        self.downloadControlButton.title = _localMapIndexItem.title;
+        if ([self isKindOfClass:OAMapDownloadController.class])
```

Comment:
Prefer adding methods to extend the behavior of a class. Here, for example, you can create a method that returns a boolean to decide whether to show region title or the Download string.